src/modules/person/domain/person-time-limit-info.service.ts

```diff
@@ -41,6 +45,36 @@ export default class PersonTimeLimitService {
             lockInfos.push(new PersonTimeLimitInfo(TimeLimitOccasion.NO_KONTEXTE, noKontexteDeadline));
         }
 
-        return lockInfos;
+        const personenKontexte: Personenkontext<true>[] = await this.dBiamPersonenkontextRepo.findByPerson(personId);
+
+        const timeLimitInfosPromises: Promise<PersonTimeLimitInfo | null>[] = personenKontexte.map(
+            async (personenKontext: Personenkontext<true>) => {
+                if (personenKontext.befristung != null) {
+                    const personenKontextExpires: Date = new Date(personenKontext.befristung);
```

Comment:
Refrain from using loose comparisons. Also comparing against `null` can miss `undefined`
```suggestion
            async (personenKontext: Personenkontext<true>) => {
                if (!!personenKontext.befristung) {
                    const personenKontextExpires: Date = new Date(personenKontext.befristung);
```